### PR TITLE
inform users, they can use new API using interpreter

### PR DESCRIPTION
**Pull request by petrus-v**
This is possible since PR #37 was merged

### CODE REVIEW

**Comment by lmignon** [comment]
:+1:

File: doc/configuration.rst [hunk 1]
```diff
@@ -932,14 +932,16 @@ The recipe will automatically create a python interpreter with a
...
-       session.open(db=DATABASE_NAME)
+        session.open(db=DATABASE_NAME)
+    or, to use the database from the buildout part config:
+        session.open()
+    All other options from buildout part config do apply.
     Then you can issue commands such as
-       session.registry('res.users').browse(session.cr, 1, 1)
-
+            session.registry('res.users').browse(session.cr, 1, 1)
+    Or using new api with odoo >= v8
```
Comment (bwrsandman) [inline doc/configuration.rst]: Could we only should this message if the server version is greater than 7? openerp.release is a good place to look

**Comment by lepistone** [comment]
:+1: thanks

**Comment by petrus-v** [comment]
Now it will display new api message only on v >=8 as suggest by @bwrsandman.

Double check on v7 and v8.

File: anybox/recipe/odoo/server.py [hunk 2]
```diff
@@ -420,9 +420,14 @@ def _install_interpreter(self):
...
             ""
-            "    print('Then you can issue commands such as')",
-            "    print(\"    "
-            "    session.registry('res.users').browse(session.cr, 1, 1)\")"
+            "    print('Then you can issue commands such as:')",
+            "    print(\""
+            "    session.registry('res.users').browse(session.cr, 1, 1)\")",
+            "    from openerp import release",
+            "    from anybox.recipe.odoo.utils import major_version",
+            "    if major_version(release.version)[0]:",
```
Comment (bwrsandman) [inline anybox/recipe/odoo/server.py]: Are you missing a `>= 8`?

Comment (petrus-v) [inline anybox/recipe/odoo/server.py]: oh yes, you're right, and I forgot to change branch before test against v7

**Comment by bwrsandman** [comment]
:+1:

**Pull request merged**